packages/metal-quartz-collapse/src/MetalQuartzCollapse.js

```diff
@@ -233,11 +229,11 @@ class MetalQuartzCollapse extends State {
 	 * @public
 	 */
 	toggle() {
-		this.transitionEventHandler_.add(
-			dom.delegate(document, this.transitionEnd, `.${this.transitionClasses}`, this.handleTransitionEnd_.bind(this))
-		);
+		const {collapsed, content, transitionEnd} = this;
+
+		dom.once(content, transitionEnd, this.handleTransitionEnd_.bind(this));
```

Comment:
The only problem I see with using `dom.once` is that multiple transitionend handlers are added if you click the toggle before the first transition has ended. Not sure if this will cause issues in the future due to multiple transitionends firing.